pythermalcomfort/classes_return.py

```diff
@@ -637,6 +637,22 @@ def __getitem__(self, item):
         return getattr(self, item)
 
 
+@dataclass(frozen=True)
+class WorkCapacity:
+    """Dataclass to represent work loss.
+
+    Attributes
+    ----------
+    capacity : float or list of floats
+        Work capacity affected by heat.
+    """
+
+    capacity: Union[float, npt.ArrayLike]
```

Comment:
in the other classess I am using `Union[float, list[float]]` but I guess what you implemented is better